### PR TITLE
Spring Boot war on Liberty - Passing configuration at runtime

### PR DESCRIPTION
**Pull request by Hemankita**

### CODE REVIEW

File: docs/spring-on-liberty/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Passing configuration at runtime to SprinBoot WAR running on WebSphere Liberty
```
Comment (osowski) [inline docs/spring-on-liberty/README.md]: typo in the title - `SpringBoot`

File: docs/spring-on-liberty/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <img src="images/springbootwar.png" height=500 width=500>
+</p>
+
+Run `./server run` to start your application. When you do this, your application will pick up the configuration values from the below yaml which is a part of the application and this is packaged along with the application in the `war` file.
+
+<p align="center">
+    <img src="images/configvalues.png" height=400 width=500>
+</p>
+
+If you want to pass the configuration values other than the ones specified in the above file, you need to override these values by passing them externally. You can do this by placing the values in `jvm.options`. You can create a file names `jvm.option` under the `shared` folder that comes with the server as below.
```
Comment (osowski) [inline docs/spring-on-liberty/README.md]: Typo in the 3rd sentence - "create a file `named`..."

File: docs/spring-on-liberty/README.md (hunk 1 above)
Comment (Hemankita) [reply docs/spring-on-liberty/README.md]: Corrected

File: docs/spring-on-liberty/README.md (hunk 2 above)
Comment (Hemankita) [reply docs/spring-on-liberty/README.md]: Corrected

**Pull request merged**